jobboard/models.py

```diff
@@ -98,11 +110,20 @@ def get_jobs_by_work_model(cls, work_model):
         return set(cls.objects.filter(work_from=work_model))
 
     @classmethod
-    def get_jobs_by_company_name(cls, company_name):
+    def get_jobs_by_company_name(cls, companies_names_as_string):
         """
-        Allows students to search for a job based on the company
+        Allows students to search for a job based on the companies
         """
-        return set(cls.objects.filter(company=Company.objects.get(name=company_name)))
+        jobs_to_return = set()
+        company_names = companies_names_as_string.split()
```

Comment:
Similar issue with the split on the cities function, a company like "Red Hat" would split into to different companies "Red" and "Hat"